static/main.js

```diff
@@ -14,7 +14,7 @@ function init() {
     input.addEventListener("input", reprocess, false);
     permalink.addEventListener("click", setSearch);
 
-    const options = {displayMode: true, throwOnError: false};
+    const options = {displayMode: true, throwOnError: true};
```

Comment:
This feels like a breaking change.  What's the rationale for this change?

Reply:
@ylemkimon As `yarn start` is for the debugging, I think it's better to throw on error and debug on the developer console.